### DISCUSSION TITLE
Cannot read properties of undefined (reading 'plugins')

### DISCUSSION THREAD

**Discussion by rochabd** [gatsbyjs/gatsby#37125] [Help]
Hello I am trying to create gatsby site.  I go through the prompts, but I get this error: Cannot read properties of undefined (reading 'plugins')

could someone provide some suggestions to resolve this issue.

**Comment by LekoArts** [comment]
Are you using `npm init gatsby` or `gatsby new`? If the latter, updating your CLI could help (with `npm install -g gatsby-cli`)

**Comment by rochabd** [comment]
thanks. it works.
